netlify.toml

```
@@ -1,3 +1,9 @@
 [build]
   publish = "_site/"
   command = "npm run prod && jekyll build"
+
+[[redirects]]
+  # Previously, Doxygen docs were hosted under "m.vtk.org/documentation/",
+  # so provide a redirect to transition any old links
+  from = "/documentation/*"
+  to = "https://docs-m.vtk.org/documentation/:splat"
```

```suggestion
  to = "https://docs-m.vtk.org"
```

---

```suggestion
  to = "https://docs-m.vtk.org/:splat"
```

I think we want instead, so any links to internal pages will still work. `:splat` injects whatever was matched by `*`.

---

+1